internal/services/applications/application_data_source.go

```diff
@@ -252,6 +349,35 @@ func applicationDataSource() *schema.Resource {
 				Computed:    true,
 			},
 
+			"single_page_application": {
```

Comment:
if this is a list shouldn't it be plural?
```suggestion
			"single_page_applications": {
```

Reply:
This block is intended to contain the settings for the application when it's functioning as a single page app (the API container is just `spa` but I felt it needed expanding)